packages/sdk-keysigner/index.ts

```diff
@@ -78,6 +78,62 @@ export class BrowserKeySigner implements CredentialSigner<KeyAssertion> {
     ])
   }
 
+  toHex = (buffer: ArrayBuffer): string => {
+    const view = new Uint8Array(buffer)
+    let hexString = ""
+    for (const byte of view) {
+      const hexByte = byte.toString(16)
+      hexString += hexByte.padStart(2, "0")
+    }
+    return hexString.toLowerCase()
+  }
```

Comment:
nit: maybe this can be extracted to a util outside of the class ?

Reply:
also we already take a dependency on browser compatible [buffer](https://www.npmjs.com/package/buffer) which I can do this. unless we don't want to have that dependency anymore.

Reply:
I wanted to avoid this dependency (less dependency = less supply chain attack surface) but if we think it would be too difficult to maintain without it I can add it.

Reply:
I put the function in utils.